### PR TITLE
Benchmark `cumulus-pallet-xcmp-queue` and white list `Treasury Account` in benchmark test

### LINKED ISSUE #537: Benchmark `cumulus_pallet_xcmp_queue`
After v0.9.18, this pallet introduced a new associated type `WeightInfo`, so we need to benchmark it.
Our CI need to support it.

### PR DESCRIPTION
**Pull request by Dengjianping**
Signed-off-by: Dengjianping <djptux@gmail.com>

Closes #537

Have to wait #481 to be merged.

Previously, we didn't include `pallet-treasury`, so this storage was not in this white list.
But now calamari-runtime has included this pallet, https://github.com/paritytech/polkadot/blob/master/runtime/kusama/src/lib.rs#L2188

### CODE REVIEW

**Comment by stechu** [comment]
@Dengjianping  How is this PR going? Still ready to schedule in 3.2.0?

**Comment by Dengjianping** [comment]
> 

Yes, we have to benchmark this pallet.

File: .github/workflows/generate_calamari_weights_files.yml [hunk 1]
```diff
@@ -236,6 +236,14 @@ jobs:
...
               name: pallet_asset_manager
             iterations: 20
+          -
+            extrinsic:
+              id: '*'
+              name: cumulus_pallet_xcmp_queue
+            pallet:
+              id: cumulus_pallet_xcmp_queue
+              name: cumulus_pallet_xcmp_queue
+            iterations: 20
```
Comment (ghzlatarev) [inline .github/workflows/generate_calamari_weights_files.yml:246]: what about in Dolphin ?

Comment (Dengjianping) [reply .github/workflows/generate_calamari_weights_files.yml:246]: Updated.

**Review by ghzlatarev — DISMISSED** [review]

**Review by ghzlatarev — APPROVED** [review]

File: runtime/calamari/src/lib.rs [hunk 2]
```diff
@@ -919,29 +933,13 @@ impl_runtime_apis! {
 				hex_literal::hex!("26aa394eea5630e07c48ae0c9558cef70a98fdbe9ce6c55837576c60c7af3850").to_vec().into(),
 				// System Events
 				hex_literal::hex!("26aa394eea5630e07c48ae0c9558cef780d41e5e16056765bc8461851072c9d7").to_vec().into(),
+				// Treasury Account
+				hex_literal::hex!("26aa394eea5630e07c48ae0c9558cef7b99d880ec681799c0cf30e8886371da95ecffd7b6c0f78751baa9d281e0bfa3a6d6f646c70792f74727372790000000000000000000000000000000000000000").to_vec().into(),
```
Comment (Garandor) [inline runtime/calamari/src/lib.rs:937]: why is this change in a benchmarking PR?

File: runtime/dolphin/src/lib.rs [hunk 3]
```diff
@@ -908,29 +922,14 @@ impl_runtime_apis! {
 				hex_literal::hex!("26aa394eea5630e07c48ae0c9558cef70a98fdbe9ce6c55837576c60c7af3850").to_vec().into(),
 				// System Events
 				hex_literal::hex!("26aa394eea5630e07c48ae0c9558cef780d41e5e16056765bc8461851072c9d7").to_vec().into(),
+				// Treasury Account
+				hex_literal::hex!("26aa394eea5630e07c48ae0c9558cef7b99d880ec681799c0cf30e8886371da95ecffd7b6c0f78751baa9d281e0bfa3a6d6f646c70792f74727372790000000000000000000000000000000000000000").to_vec().into(),
```
Comment (Garandor) [inline runtime/dolphin/src/lib.rs:926]: same here

**Review by Garandor — APPROVED** [review]
Please change the PR title and description so the hidden change is clearly stated.
Otherwise LGTM!

File: runtime/calamari/src/lib.rs (hunk 2 above)
Comment (Dengjianping) [reply runtime/calamari/src/lib.rs:937]: Previously, we didn't include `pallet-treasury`, so this storage was not in this white list.
But now calamari-runtime has included this pallet, https://github.com/paritytech/polkadot/blob/master/runtime/kusama/src/lib.rs#L2188

**Comment by Dengjianping** [comment]
> Please change the PR title and description so the hidden change is clearly stated. Otherwise LGTM!

Sorry, I just forgot to update it.

**Pull request merged**